pocket_coffea/lib/gen_objects.py

```diff
@@ -0,0 +1,39 @@
+import awkward as ak
+import numpy as np
+
+def Gen_leptons(events, lepton_flavour, params):
```

Comment:
it may just be my preference, but what about renaming the function `getGenLeptons`?   `GenLeptons` sounds like a class a not a function :)

Reply:
Sure, this change is fine with me. 
However, unless there are other suggestions, I would prefer to change it in my next PR. 
I have modified my code on top of this PR, and don't not want to revert for a small fix like this.

Reply:
sure nothing blocking! Thanks